server/serverMethods.js

```diff
@@ -668,6 +669,7 @@ function ServerMethods(aLogLevel, aModules) {
                 var dialedNumberInfo = {};
                 dialedNumberInfo.sessionId = sipCallData.sessionId;
                 dialedNumberInfo.connectionId = sipCallData.connectionId;
+                dialedNumberInfo.googleIdToken = googleIdToken;
```

Comment:
I'm glad this server persistence thing I did ended up getting used for something!